# Query for the meshes of specific entities
string[] entity_ids
---
EntityMeshAndAreas[] entity_geometries

# Empty if no errors
string error_msg
